Merge_Sort_3_Way/Merge_Sort_3_Way.c

```diff
@@ -0,0 +1,123 @@
+#include<stdio.h>
+//#include<stdlib.h>
+
+void mergesort(long int *arr1, int p , int q ,long int *arr2);
+int merge(long int *arr1,int l,int m1, int m2, int h, long int *arr2 );
+
+int main()
+{ 
+
+  int length;
+
+  long int n;
+  printf("Number of elements in the array : ");
+  scanf("%ld",&n);
+  
+  long int* arr1 = (long int *)malloc(n * sizeof(long int));
+  long int* arr2=(long int *)malloc (n * sizeof(long int ));
+  
+  int i=0;
+  long int num;
+
+  
+
+  while(i<n)
+  {
+    printf("Enter the %d number : ", i + 1);
+    scanf("%ld",&num);
+	  arr1[i]=num;
+	  arr2[i]=num;
+	  i++;
```

Comment:
Fix indentation - 4 spaces in place of tab, 8 spaces in place of 2 tabs, and so on...